ipatests/test_integration/test_epn.py

```diff
@@ -708,6 +712,7 @@ def test_mailtest_dry_run(self):
             raise AssertionError('--mail-test and --dry-run aren\'t supposed '
                                  'to succeed')
 
+    @skip_if_fips()
```

Comment:
I don't think that this skip is needed, the run from Nightly-STIG-Tier2/RHEL9.3/2023-09-15_00-42 shows a successful test.